metadeploy/api/admin.py

```diff
@@ -213,13 +213,16 @@ class PreflightResult(AdminHelpTextMixin, admin.ModelAdmin, PlanMixin):
 
 @admin.register(Product)
 class ProductAdmin(MetadeployTranslatableAdmin):
-    list_display = ("title", "category", "order_key")
+    list_display = ("title", "category", "order_key", "is_listed")
+    list_filter = ("is_listed", "category")
     search_fields = ("translations__title", "translations__description")
 
 
 @admin.register(ProductCategory)
-class ProductCategoryAdmin(admin.ModelAdmin):
-    list_display = ("title", "order_key")
+class ProductCategoryAdmin(MetadeployTranslatableAdmin):
+    list_display = ("title", "order_key", "is_listed")
+    list_filter = ("is_listed",)
```

Comment:
Opportunity: `list_editable = ("order_key", "is_listed")` for easy bulk edits. It might be a good idea to make `order_key` list-editable everywhere (feel free to ignore, it's not related to this PR)